pkg/archive/archive_linux.go

```diff
@@ -9,9 +9,17 @@ import (
 
 	"github.com/containers/storage/pkg/idtools"
 	"github.com/containers/storage/pkg/system"
+	"github.com/containers/storage/pkg/unshare"
 	"golang.org/x/sys/unix"
 )
 
+func getWhiteoutXattrName() string {
```

Comment:
Shouldn't you use the getXattrName function?

Reply:
it is in a different package

Reply:
Sure, but should we make it public and use it?

Reply:
the best place seems `drivers/overlay`.  That will cause to have a dependency on it from `pkg/archive` and `drivers/copy`.

Do you have any suggestions?  `pkg/unshare`?

Reply:
I was thinking just put it in archive?

Reply:
I've tried in archive but it looks out of place.

What do you think about the current implementation and having the dependency only for `pkg/archive`?